index.js

```diff
@@ -399,6 +402,35 @@
     function(x) { return $.FiniteNumber.test(x) && Number(x) !== 0; }
   );
 
+  //  Integer :: Type
+  $.Integer = $.NullaryType(
+    'sanctuary-def/Integer',
+    function(x) {
+      return $.ValidNumber.test(x) &&
+        Math.floor(x) === Number(x) &&
+        x >= MIN_SAFE_INTEGER  &&
+        x <= MAX_SAFE_INTEGER;
+    }
```

Comment:
Yeah, that's how I had it originally and then I saw in the `Integer` definition in `test\index.js` that the object constructor was used so I changed it.